src/tui/proc-filter.go

```diff
@@ -0,0 +1,68 @@
+package tui
+
+import (
+	"github.com/gdamore/tcell/v2"
+	"github.com/rivo/tview"
+)
+
+func (pv *pcView) showProcFilter() {
+	const fieldWidth = 50
+	f := tview.NewForm()
+	f.SetCancelFunc(func() {
+		pv.pages.RemovePage(PageDialog)
+	})
+	f.SetItemPadding(1)
+	f.SetBorder(true)
+	f.SetFieldBackgroundColor(tcell.ColorLightSkyBlue)
+	f.SetFieldTextColor(tcell.ColorBlack)
+	f.SetButtonsAlign(tview.AlignCenter)
+	f.SetTitle("Search Process")
+
+	f.AddInputField("Search For", pv.logsText.getSearchTerm(), fieldWidth, nil, nil)
```

Comment:
```golang
pv.logsText.getSearchTerm()
```
Looks like a copy-paste from the log search 😉
It should be the current search term if wasn't reset.